src/components/alarm/AlarmController.cpp

```diff
@@ -15,10 +15,13 @@
     You should have received a copy of the GNU General Public License
     along with this program.  If not, see <https://www.gnu.org/licenses/>.
 */
-#include "AlarmController.h"
+#include "components/alarm/AlarmController.h"
 #include "systemtask/SystemTask.h"
-#include "app_timer.h"
-#include "task.h"
+// nrf5 sdk
+#include <app_timer.h>
+// FreeRTOS library
+#include <task.h>
+// standard libraries
```

Comment:
I don't think the comments are necessary.

Reply:
They add a bit of structure for future contributions to keep the include order. And it would have been helpful for my first exploration of the source code. For example I had no idea, that `<task.h>` is part of FreeRTOS. With the comment this is explicitly listet

Reply:
Sounds like a dev documentation addition instead of each file.